lib/ash/resource/change/cascade_update.ex

```diff
@@ -204,9 +204,21 @@ defmodule Ash.Resource.Change.CascadeUpdate do
         return_notifications?: opts.return_notifications?
       )
 
+    context = Map.merge(relationship.context || %{}, %{cascade_destroy: true})
+
     case related_query(data, opts) do
       {:ok, query} ->
-        Ash.bulk_update(query, action.name, inputs, context_opts)
+        Ash.bulk_update(
+          query,
+          action.name,
+          inputs,
+          Keyword.update(
+            context_opts,
+            :context,
+            relationship.context || %{},
+            &Map.merge(&1, relationship.context || %{})
```

Comment:
is there a reason that `cascade_destroy: true` is not passed into this bulk_destroy call?

Reply:
No reason, makes sense to include it there (just using the `context`  you defined above)